app/src/pages/dataset/RunExperimentButton.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { ReactNode, useCallback, useMemo, useState } from "react";
+
+import {
+  Button,
+  Dialog,
+  DialogContainer,
+  Icon,
+  Icons,
+  Text,
+  View,
+} from "@arizeai/components";
+
+import { PythonBlockWithCopy } from "@phoenix/components/code/PythonBlockWithCopy";
+import { useDatasetContext } from "@phoenix/contexts/DatasetContext";
+
+const INSTALL_PHOENIX_PYTHON = `!pip install arize-phoenix==${window.Config.platformVersion}`;
+// TODO: plumb though session URL from the backend
+const SET_BASE_URL_PYTHON =
+  `import os\n` +
+  `# Set the phoenix collector endpoint. Commonly http://localhost:6060 \n` +
+  `os.environ["PHOENIX_COLLECTOR_ENDPOINT"] = "<your-phoenix-url>"`;
+const TASK_PYTHON =
+  `from phoenix.datasets.types import Example\n` +
+  `# Define your task\n` +
+  `# Typically should be an LLM call or a call to your application\n` +
+  `def my_task(example: Example) -> any:\n` +
+  `    return f"Hello {example.input}"`;
+
+const EVALUATOR_PYTHON =
+  `# Define an evaluator. This just an example.\n` +
+  `def exact_match(input, output) -> float:\n` +
+  `    return 1.0 if output is f"Hello {input}" else 0.0\n\n` +
```

Comment:
```suggestion
  `    return 1.0 if output == f"Hello {input}" else 0.0\n\n` +
```

Usually the input is a dictionary, I think.

Reply:
yeah this might be wrong.